### PR TITLE
Update demo app for release 4.0.2

### PR DESCRIPTION
**Pull request by zhangcc01**
## Description

Setup DemoApp for testing for React Native 4.0.2 release.

### CODE REVIEW

File: DemoApp/android/app/src/main/java/com/demoapp/MainApplication.java [hunk 1]
```diff
@@ -36,6 +37,7 @@ public boolean getUseDeveloperSupport() {
         protected List<ReactPackage> getPackages() {
             return Arrays.asList(
                     new MainReactPackage(),
+            new RNScreensPackage(),
```
Comment (MatkovIvan) [inline DemoApp/android/app/src/main/java/com/demoapp/MainApplication.java:40]: Please format it (align)

Comment (zhangcc01) [reply DemoApp/android/app/src/main/java/com/demoapp/MainApplication.java]: @MatkovIvan Resolved.

File: DemoApp/ios/DemoApp/AppDelegate.m [hunk 2]
```diff
@@ -24,6 +24,7 @@
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [AppCenterReactNative register];  // Initialize AppCenter
```
Comment (russelarms) [inline DemoApp/ios/DemoApp/AppDelegate.m:27]: Duplicated call:
```suggestion
```

**Review by russelarms — CHANGES_REQUESTED** [review]

File: DemoApp/ios/DemoApp/AppDelegate.m (hunk 2 above)
Comment (zhangcc01) [reply DemoApp/ios/DemoApp/AppDelegate.m]: @russelarms Resolved.

**Review by DmitriyKirakosyan — APPROVED** [review]

**Review by russelarms — APPROVED** [review]

**Pull request merged**